guild/tests/ls-cmd.md

```diff
@@ -104,6 +104,7 @@ List all files with `--all`.
       .guild/attrs/stopped
       .guild/attrs/user
       .guild/attrs/user_flags
+      .guild/attrs/vcs_commit
```

Comment:
I don't understand why this addition came about. I'll investigate it once the rest of the errors are worked out.